dns.tf

```diff
@@ -46,6 +46,7 @@ variable "subdomain" {
     "climate.usegalaxy.eu",
     "nanopore.usegalaxy.eu",
     "metabolomics.usegalaxy.eu",
+    "microbiome.usegalaxy.eu",
```

Comment:
Can you add this to the end of the list please?

Reply:
done